include/aos/common/tools/allocator.hpp

```diff
@@ -11,8 +11,8 @@
 #include <assert.h>
```

Comment:
I think better create a Set class for allocations

Reply:
In general yes, but from current allocator implementation view, there is no advantages using set. Also, list has potentially more use cases in the future.